packages/k8s/APKBUILD

```diff
@@ -0,0 +1,44 @@
+# Contributor:
+# Maintainer:
+pkgname=k8s
+pkgver="v1.8.3"
+pkgrel=0
+pkgdesc="Kubernetes kubeadm"
+url="https://kubernetes.io/"
+arch="armhf"
+license="Apache License 2.0"
+depends="ebtables ethtool socat"
+makedepends=""
+install=""
+subpackages=""
+source="
+        kubeadm::https://storage.googleapis.com/kubernetes-release/release/${pkgver}/bin/linux/arm/kubeadm
+        kubelet::https://storage.googleapis.com/kubernetes-release/release/${pkgver}/bin/linux/arm/kubelet
+        kubectl::https://storage.googleapis.com/kubernetes-release/release/${pkgver}/bin/linux/arm/kubectl
+        kubelet.initd
+        kubelet.confd
+        "
+builddir="$srcdir/"
+options="!check"
+
+build() {
+        return 0
+}
+
+package() {
+
+        install -m755 -D "$srcdir"/kubelet.initd \
+                "$pkgdir"/etc/init.d/kubelet
+        install -m644 -D "$srcdir"/kubelet.confd \
+                "$pkgdir"/etc/conf.d/kubelet
+
```

Comment:
Do you need to include the `kubelet` startup script in a particular runlevel e.g. `default`?

Reply:
Kubelet should be started manually when initialising master or adding a worker. I think it shouldn't be started on boot as some nodes might not be either a master or a worker. We'll figure out, later on, how to automatically add workers.